code/dqe/savedata.q

```diff
@@ -0,0 +1,18 @@
+\d .dqe
+savedata:{[dir;pt;tabname]                                                                                       /- function to save data persistently
+  .lg.o[`dqe;"Saving ",(string tabname)," data to ", string dir];
+  .[
+    upsert;
+    (` sv .Q.par[dir;pt;tabname],`;.Q.en[dir;r:0!.save.manipulate[tabname;`. tabname]]);                         /- upsert data to partition
+    {[e] .lg.e[`savedata;"Failed to save dqe data to disk : ",e];`e}
```

Comment:
This will return the symbol e and not the error message as a symbol. But I think this is supposed to throw an error? If so:
```suggestion
    {[e] .lg.e[`savedata;"Failed to save dqe data to disk : ",e];'e}
```
Else use `` `$`` to convert to symbol.